CHANGELOG.md

```diff
@@ -4,6 +4,257 @@ All notable changes to Router will be documented in this file.
+
+By [@BrynCooke](https://github.com/BrynCooke) in https://github.com/apollographql/router/pull/5375
+
+### Log metrics overflow as a warning rather than an error ([Issue #5173](https://github.com/apollographql/router/issues/5173))
+
+If a metric has too high a cardinality, the following is displayed as a warning instead of an error:
+
+`OpenTelemetry metric error occurred: Metrics error: Warning: Maximum data points for metric stream exceeded/ Entry added to overflow`
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5287
+
+### Add support of `response_context selector` when in errors ([PR #5288](https://github.com/apollographql/router/pull/5288))
+
+Provides the ability to configure custom instruments. For example:
+
+```yaml
+http.server.request.timeout:
+  type: counter
+  value: unit
+  description: "request in timeout"
+  unit: request
+  attributes:
+    graphql.operation.name:
+      response_context: operation_name
+  condition:
+    eq:
+    - "request timed out"
+    - error: reason
+```
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5288
+
+### Inaccurate `apollo_router_opened_subscriptions` counter ([PR #5363](https://github.com/apollographql/router/pull/5363))
+
+Fixes the `apollo_router_opened_subscriptions` counter which previously only incremented. The counter now also decrements.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5363
+
+## 📃 Configuration
+
+### Rename trace telemetry selector ([PR #5337](https://github.com/apollographql/router/pull/5337))
+
+[v1.48.0](https://github.com/apollographql/router/releases/tag/v1.48.0) introduced the `apollo` `trace_id` selector. `trace_id` is a misnomer for this metric, since the selector actually represents a GraphOS Studio operation ID. To access this selector, use `studio_operation_id`:
+
+```yaml
+telemetry:
+  instrumentation:
+    spans:
+      router:
+        "studio.operation.id":
+            studio_operation_id: true
+```
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5337
+
+### Set Apollo metrics generation mode to `new` by default  ([PR #5265](https://github.com/apollographql/router/pull/5265))
+
+Changes the default value of `experimental_apollo_metrics_generation_mode` to `new`. All metrics are showing that identical signatures are being generated in this mode.
+
+By [@bonnici](https://github.com/bonnici) in https://github.com/apollographql/router/pull/5265
+
+## 🛠 Maintenance
+
+### Skip GraphOS tests when Apollo key not present ([PR #5362](https://github.com/apollographql/router/pull/5362))
+
+Some tests require `APOLLO_KEY` and `APOLLO_GRAPH_REF` to execute successfully.
+These are now skipped if these env variables are not present.
+
+By [@BrynCooke](https://github.com/BrynCooke) in https://github.com/apollographql/router/pull/5362
+
```

Comment:
Is this meaningful to the end-user, @BrynCooke?

Reply:
Added a suggestion, it is meaningful to external users contributing to the router.